tests/unit/vertical-bar-test.js

```diff
@@ -138,13 +142,74 @@ test('Stacked bar chart data is sorted correctly', function(assert) {
   });
 });
 
-test('Stacked bars are grouped correctly', function(assert) {
-  var labelIDMapping;
-  this.subject(stackedBarContent);
+// https://github.com/Addepar/ember-charts/issues/172
+//
+// The root cause of problem 2 was that when some data groups have fewer data than others,
+// the bars for the smaller groups got assigned the wrong style classes.
+// The classes determine which bars get highlighted when you mouse over the bar label
+// in the legend.
+//
+test('Bug 172 Problem 2: Bars get style classes corresponding to their bar label, ' +
+     'regardless of what bar group they belong to',
+function(assert) {
+  this.subject({data: labelClassMappingTestData});
```

Comment:
This test is almost identical to the next one except for the `stackBars` toggle; could we extract the logic to just run the same function twice with a different `stackBars` value?